user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -412,27 +440,35 @@ define([
       }
 
       fileButtons.canDownload(this.get("canDownload"));
-      //TODO handle file button press
+
       var filePath = clickedFile.obj.attr("path");
-      filePath = Encoder.encodeRepositoryPath(filePath);
+      const isRepoPath = isRepositoryPath(filePath);
+      fileButtons.enableButtons(isRepoPath);
 
       //Ajax request to check write permissions for file
-      $.ajax({
-        url: CONTEXT_PATH + 'api/repo/files/' + FileBrowser.encodePathComponents(filePath) + '/canAccessMap',
-        type: "GET",
-        beforeSend: function (request) {
-          request.setRequestHeader('accept', 'application/json');
-        },
-        data: {"permissions": "1|2"}, //check write and delete permissions for the given file
-        async: true,
-        cache: false,
-        success: function (response) {
-          fileButtons.updateFilePermissionButtons(response);
-        },
-        error: function (response) {
-          fileButtons.updateFilePermissionButtons(false);
-        }
-      });
+      if( filePath.charAt(0) == "/" ) {
```

Comment:
Test should use the new`isRepoPath` variable.

Reply:
ah, also, I forgot to push my latest changes for `_isRepositoryPath` -> `isRepositoryPath`